stable/sematext-docker-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```

Comment:
For DaemonSet could you use apps/v1beta2 as the apiVersion? extensions/v1beta1 will not be around much longer as they are deprecated.

Reply:
Fixed, you are right.